.travis.yml

```diff
@@ -11,7 +11,7 @@ android:
   - sys-img-armeabi-v7a-android-15
 env:
   - ANDROID_TARGET=android-15 ANDROID_ABI=armeabi-v7a
-script: ./gradlew build
+script: ./gradlew build assembleAndroidTest
```

Comment:
If `after_success` returns an error it does not fail the build. I added guava, exceeded method count and the build still passed.